src/common/commonutils/FileUtils.c

```diff
@@ -1199,6 +1198,16 @@ int CheckTextIsNotFoundInFile(const char* fileName, const char* text, char** rea
     return result;
 }
 
+static bool IsValidGrepArgument(const char* text)
+{
+    return IsValidDaemonName(text);
+}
+
+static bool IsValidCommentCharacter(char c)
+{
+    return ((0 == isalnum(c)) && ('#' != c)) ? false : true;
+}
+
 int CheckMarkedTextNotFoundInFile(const char* fileName, const char* text, const char* marker, char commentCharacter, char** reason, void* log)
 {
     const char* commandTemplate = "grep -v '^%c' %s | grep %s";

```

Comment:
How about files that start with spaces and then a comment character? Is this a possibility?

Reply:
Not for us. If it were, we'd have to consider space a comment character and then comment sequences of multiple characters, etc.

Reply:
Not that I am aware of.